src/cli/command.js

```diff
@@ -96,7 +110,7 @@ Command.prototype.action = _.wrap(Command.prototype.action, function (action, fn
     const ret = fn.apply(this, args);
     if (ret && typeof ret.then === 'function') {
       ret.then(null, function (e) {
-        console.log('FATAL CLI ERROR', e.stack);
+        console.log('FATAL CLI ERROR', e.stack);
```

Comment:
It looks like this was changed to a non breaking space, any context on the change?

Reply:
I changed it to a regular space from a https://www.compart.com/en/unicode/U+00A0 character. I thought I'd clean it up while I'm there - although, maybe this was an intentional non-breaking space?
This was the original: 
<img width="545" alt="Screenshot 2023-05-26 at 13 14 26" src="https://github.com/elastic/kibana/assets/4738868/34a32807-e1ce-4bf0-995d-0286a4592552">

Reply:
I don't see anything in the commit history that alludes to it, but it wouldn't surprise me.  Seems fine either way.